<h1 style="text-align:center;">single cell perturb paper repo</h1>

|ID|paper name|paper link|code link|publish date|journal|
|---|---|---|---|---|---|
|1|Dr.VAE: improving drug response prediction via modeling of drug perturbation effects|[paper](https://doi.org/10.1093/bioinformatics/btz158)|[code](https://github.com/rampasek/DrVAE)|2019.03.08|bioinformatics|
|2|scPreGAN, a deep generative model for predicting the response of single-cell expression to perturbation|[paper](https://doi.org/10.1093/bioinformatics/btac357)|[code](https://github.com/JaneJiayiDong/scPreGAN)|2022.05.26|bioinformatics|
|3|scGen predicts single-cell perturbation responses|[paper](https://www.nature.com/articles/s41592-019-0494-8)|[code](https://github.com/theislab/scgen)|2019.07.29|nature method|
|4|Predicting transcriptional responses to novel chemical perturbations using deep generative model for drug discovery|[paper](https://doi.org/10.1038/s41467-024-53457-1)|[code](https://github.com/Perturbation-Response-Prediction/PRnet)|2024.10.26|nature communications|
|5|Machine learning for perturbational single-cell omics|[paper](https://doi.org/10.1016/j.cels.2021.05.016)||2021.06.16|cell systems|
|6|limma powers differential expression analyses for RNA-sequencing and microarray studies|[paper](https://doi.org/10.1093/nar/gkv007)||2015.04.20|Nucleic Acids Research|
|7|Predicting transcriptional outcomes of novel multigene perturbations with GEARS|[paper](https://doi.org/10.1038/s41587-023-01905-6)|[code](https://github.com/snap-stanford/GEARS)|2023.08.13|nature biotechnology|
|8|Predicting Cellular Responses to Novel Drug Perturbations at a Single-Cell Resolution|[paper](https://openreview.net/forum?id=vRrFVHxFiXJ)|[code](https://github.com/theislab/chemCPA)|2022.11.01|NeurIPS 2022|
|9|Learning single-cell perturbation responses using neural optimal transport|[paper](https://doi.org/10.1038/s41592-023-01969-x)|[code](https://github.com/bunnech/cellot)|2023.09.28|nature methods|
